From 4d27bfce52d597fcbf31268876703176da7c19a0 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Wed, 12 Sep 2012 13:10:28 -0300
Subject: [RHEL6 qemu-kvm PATCH 09/14] usb-redir: Get rid of unused
 async-struct dev member

RH-Author: Hans de Goede <hdegoede@redhat.com>
Message-id: <1347455431-11260-10-git-send-email-hdegoede@redhat.com>
Patchwork-id: 41855
O-Subject: [RHEL-6.4 qemu-kvm PATCH 09/12] usb-redir: Get rid of unused async-struct dev member
Bugzilla:
RH-Acked-by: Alon Levy <alevy@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Christophe Fergeau <cfergeau@redhat.com>
Bugzilla-related: 805172

This is a preparation patch for completely getting rid of the async-packet
struct in usb-redir, instead relying on the (new) per ep queues in the
qemu usb core.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
Upstream-commit: 206e7f20fe7b920b362bcc02608680c5d5527f2a
Conflicts: usb-redir.c
---
 usb-redir.c | 2 --
 1 file changed, 2 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 usb-redir.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/usb-redir.c b/usb-redir.c
index d2e1bf9..cbe83dd 100644
--- a/usb-redir.c
+++ b/usb-redir.c
@@ -95,7 +95,6 @@ struct USBRedirDevice {
 };
 
 struct AsyncURB {
-    USBRedirDevice *dev;
     USBPacket *packet;
     uint32_t packet_id;
     QTAILQ_ENTRY(AsyncURB)next;
@@ -256,7 +255,6 @@ static int usbredir_write(void *priv, uint8_t *data, int count)
 static AsyncURB *async_alloc(USBRedirDevice *dev, USBPacket *p)
 {
     AsyncURB *aurb = (AsyncURB *) qemu_mallocz(sizeof(AsyncURB));
-    aurb->dev = dev;
     aurb->packet = p;
     aurb->packet_id = dev->packet_id;
     QTAILQ_INSERT_TAIL(&dev->asyncq, aurb, next);
-- 
1.7.11.4

